lib/models/IonUpdateRequestModel.ts

```diff
@@ -11,12 +13,7 @@ export default interface IonUpdateRequestModel {
   revealValue: string;
   delta: {
     updateCommitment: string,
-    patches: {
-        action: string,
-        servicesToAdd?: IonServiceModel[],
-        publicKeys?: IonPublicKeyModel[],
-        ids?: string[]
-    }[]
+    patches: (IonAddServicesActionModel | IonAddPublicKeysActionModel | IonRemoveServicesActionModel | IonRemovePublicKeysActionModel)[]
```

Comment:
FYI: I think we should have a `createReplaceRequest` at some point (non-blocking for this PR).